pyfolio/tears.py

```diff
@@ -938,6 +956,7 @@ def create_interesting_times_tear_sheet(
     return_fig : boolean, optional
         If True, returns the figure that was plotted on.
     """
+    returns, _, _ = sanitize.sanitize(returns=returns)
```

Comment:
Why not just call `sanitize_returns` directly here? (There are a few other instances of this in the functions below).

Reply:
You're right! An amazing act of mental closure on my part.